crates/shared/src/tenderly_api.rs

```diff
@@ -102,6 +102,24 @@ impl TenderlyApi for TenderlyHttpApi {
         Ok(serde_json::from_str(&body)?)
     }
 
+    fn log(&self, simulation: SimulationRequest) -> Result<()> {
+        let request_url = crate::url::join(&self.api, "simulate");
+        let simulation_url =
+            crate::url::join(&self.dashboard, "simulator/$SIMULATION_ID").to_string();
+        let body = serde_json::to_string(&simulation)?;
```

Comment:
nit: any test that this is properly formatted?

Reply:
This would only work with a test that gets ignored by default (since we need a secret and don't want to waste limited tenderly resources on tests) and those tend to be dead code anyway because people usually don't rerun them regularly AFAIK.
I tested the code manually and ran the suggested command which produced the expected output, though.